pkg/destroy/vsphere/vsphere.go

```diff
@@ -217,12 +217,14 @@ func (o *ClusterUninstaller) Run() error {
 		o.Logger.Errorln(err)
 		return err
 	}
+	o.Logger.WithField("Tag", o.InfraID).Info("Destroyed")
 
 	o.Logger.Debug("Delete tag category")
 	if err = deleteTagCategory(context.TODO(), o.RestClient, "openshift-"+o.InfraID); err != nil {
 		o.Logger.Errorln(err)
 		return err
 	}
+	o.Logger.WithField("Tag Category", "openshift-"+o.InfraID).Info("Destroyed")
```

Comment:
It is a bit easier to read log entries when the log fields do not have spaces.
```suggestion
	o.Logger.WithField("TagCategory", "openshift-"+o.InfraID).Info("Destroyed")
```